libs/feature/record/src/lib/record-metadata/record-metadata.component.ts

```diff
@@ -77,4 +82,11 @@ export class RecordMetadataComponent {
       .getFiltersForOrgs([{ name: contact.organisation }])
       .subscribe((filters) => this.searchService.updateFilters(filters))
   }
+
+  openLightbox(src: string) {
+    const lightboxInstance = basicLightbox.create(`<img src="${src}"/>`)
+    lightboxInstance.show()
+    //returns for test only
+    return lightboxInstance
```

Comment:
Could you maybe add a class name to the `.create` function instead of returning only to test it? WDYT?

Reply:
I like the approach with the lightboxInstance on the component level better :-)
Thanks for addressing this!
I will resolve the commment.